features/step_definitions/edit_support_actions_steps.rb

```diff
@@ -1,54 +1,54 @@
-Given('a resident with a need exists') do
+Given('a resident with a support action exists') do
   @contact = Contact.create!(first_name: 'Test')
-  @need = Need.create!(contact_id: @contact.id, name: 'Phone Triage', category: 'Phone triage')
+  @need = Need.create!(contact_id: @contact.id, name: 'Dog walking', category: 'Dog walking')
```

Comment:
Why was this changed to Dog walking? Was there an issue with the date field or something?